zapisy/apps/notifications/signals.py

```diff
@@ -175,3 +175,24 @@ def notify_board_members_about_voting(sender: Thesis, **kwargs) -> None:
                      NotificationType.THESIS_VOTING_HAS_BEEN_ACTIVATED, {
             'title': thesis.title
         }, target))
+
+
+@receiver(defect_modified)
+def notify_that_defect_was_modified(sender, **kwargs) -> None:
```

Comment:
W tym momencie jedyną klasą, której obiekty wysyłają ten sygnał, jest `Defect` – to pewnie nic nie zmienia, ale czy nie byłoby bardziej ostrożne doprecyzować tak, jak choćby w innych użyciach w tym module?
```suggestion
@receiver(defect_modified, sender=Defect)
def notify_that_defect_was_modified(sender: Defect, **kwargs) -> None:
```